cmd/params/params.go

```diff
@@ -750,29 +752,20 @@ func readAPIKeyFromCommand(cmdStr string) (string, error) {
 	return strings.TrimSpace(string(out)), nil
 }
 
-var extraHeartbeatsCache *[]heartbeat.Heartbeat // nolint:gochecknoglobals
```

Comment:
I still want to use a semaphore to be sure it never reads stdin twice. It can't hurt.